docs/openrpc.json

```diff
@@ -1307,7 +1321,7 @@
                 ]
             },
             "TransactionSigned": {
-                "oneOf": [
+                "anyOf": [
```

Comment:
Q: why the change?

Reply:
Oh good question and glad you asked.
It was failing, and I had no idea why 😅 until I realized that it was doing match with both , since `oneOf` was looking for either: `Transaction1559Signed` or `TransactionLegacySigned`. and since I added `v` to `Transaction1559Signed` now it was matching for both of them 😅  and hence failing.

in other words, now that we included `v` on `Transaction1559Signed` it now became a superset of `TransactionLegacySigned` and it was matching on both schemas, and since `oneOf` means that only can match `1`. and `anyOf` can match `1` or more.

Reply:
Interesting. Thanks for clarifying